test/interception.spec.js

```diff
@@ -336,6 +336,26 @@ describe('Page.route', function() {
     expect(response.ok()).toBe(true);
     expect(intercepted).toBe(true);
   });
+  // WebKit crashes. Firefox succeeds, but then fails to close.
```

Comment:
I suspect failing to close might be unrelated issue, I've seen that in FF a few times.